gadgets/trace_exec/test/integration/trace_exec_test.go

```diff
@@ -41,6 +41,7 @@ type traceExecEvent struct {
 	Error       string `json:"error"`
 	UpperLayer  bool   `json:"upper_layer"`
 	PupperLayer bool   `json:"pupper_layer"`
+	FupperLayer bool   `json:"fupper_layer"`
```

Comment:
This unit test does not test the distinction between UpperLayer and FupperLayer when running a script. This is not easy to test the two cases because we don't have a container image with an unmodified script.

But you could run a test by creating a script in `/dev/myscript.sh` and in `/bin/myscript.sh`. Since `/dev` is a tmpfs, it will never be on an overlay upper layer.

```
printf '#!/bin/sh\ncat /dev/null' > /dev/myscript.sh ; chmod +x /dev/myscript.sh ; /dev/myscript.sh
printf '#!/bin/sh\ncat /dev/null' > /bin/myscript.sh ; chmod +x /bin/myscript.sh ; /bin/myscript.sh
```
The first command should have FupperLayer=false, and the second command should have FupperLayer=true.